datafusion/datasource-parquet/src/reader.rs

```diff
@@ -314,3 +334,56 @@ impl FileMetadata for CachedParquetMetaData {
         HashMap::from([("page_index".to_owned(), page_index.to_string())])
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use datafusion_execution::cache::cache_unit::DefaultFilesMetadataCache;
+    use object_store::{memory::InMemory, path::Path, ObjectMeta, ObjectStore};
+
+    #[tokio::test]
+    async fn cached_parquet_file_reader() {
```

Comment:
Instead of a test just to test new, how about changing the existing code to call `new()` rather than constructing `CachedParquetFileReader` directly?